gsy_e_sdk/rest_device.py

```diff
@@ -1,147 +1,17 @@
 import logging
-from concurrent.futures.thread import ThreadPoolExecutor
-from typing import Dict
+from gsy_e_sdk.clients.rest_asset_client import RestAssetClient
 
-from gsy_framework.client_connections.utils import (
-    RestCommunicationMixin, retrieve_jwt_key_from_server, blocking_post_request)
-from gsy_framework.client_connections.utils import log_market_progression
-from gsy_framework.client_connections.websocket_connection import WebsocketThread
-from gsy_framework.utils import execute_function_util
+logger = logging.getLogger(__name__)
+console_handler = logging.StreamHandler()
+console_handler.setFormatter(logging.Formatter("%(levelname)s: %(message)s"))
+logger.addHandler(console_handler)
 
-from gsy_e_sdk import APIClientInterface
-from gsy_e_sdk.constants import MAX_WORKER_THREADS
-from gsy_e_sdk.utils import (
-    logging_decorator, get_aggregator_prefix,
-    domain_name_from_env, websocket_domain_name_from_env,
-    simulation_id_from_env, get_configuration_prefix, log_trade_info)
-from gsy_e_sdk.websocket_device import DeviceWebsocketMessageReceiver
+logger.warning(
+    "The RestDeviceClient class name will be deprecated. Please use RedisAssetClient instead.")
 
-REGISTER_COMMAND_TIMEOUT = 15 * 60
 
+class RestDeviceClient(RestAssetClient):
```

Comment:
Minor, however for future purposes, keep in mind that with this approach (moving the content of the module / class to a new one, and marking the existing module / class as deprecated) the git history will be lost (because this code was moved, I assume that we cannot backtrack to past changes in the code). Not really important and I do not think that it is required to change it though.

Reply:
How would you suggest doing it? We could in theory do the opposite (keep the content in `RestDeviceClient` and use `RestAssetclient(RestDeviceClient)`. But then we would be making future changes to a class that is deprecated, and it's a bit counter-intuitive.

Reply:
The correct way IMO would be:
1. rename the module (rest_device.py -> rest_asset.py)
2. rename RestDeviceClient->RestAssetClient
3. create a new RestDeviceClient class with a warning log 
4. commit your changes in order for git to track the renames in its history
5. create a new module called rest_device.py , and move the class RestDeviceClient there 
6. Commit

Reply:
But in your steps the `rest_asset.py` stays in the same location of the old module. What I wanted to do was to create a `clients/` package and move the new module inside. I suppose that this would still break the history.

Reply:
I'll definitely try it out next time! Thanks